### PR TITLE
Update telematic UI to list and upload ROS 2 rosbag

### PR DESCRIPTION
**Pull request by dan-du-car**
# PR Details
## Description
Create a ROS2 rosbag page to display the list of ROS2 rosbags, and their upload/processing status. 
Create a dialog to allow users to upload ROS2 rosbags.
The dialog should popup when users click on an upload button in the ROS2 rosbag list page. In the dialog, users can choose multiple files and update upload description for each file. 
Display a preview of the files to be uploaded, and a delete button for each file.
Details of the UI page refer to https://usdot-carma.atlassian.net/wiki/spaces/WFD2/pages/2707226627/Web+UI+Historical+Data+Page+Design 
Call file upload backend service to upload ROS2 rosbags and get  a list of uploaded ROS2 rosbags.
Call file upload backend service to update description of uploaded files.
Make UI button more consistent cross all pages.
Update UI logo.
Update main navigation tab text.
Update docker-compose files and environment variables with the latest parameters.
Add filters to filter the list of ROS2 rosbag table.
Add unit tests.

## Related GitHub Issue
NA

## Related Jira Key
https://usdot-carma.atlassian.net/browse/WFD-499

## Motivation and Context
Telematic Tool

## How Has This Been Tested?
Local integration testing and unit test

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: telematic_system/telematic_apps/web_app/server/file_upload_server.js [hunk 1]
```diff
@@ -62,52 +85,86 @@ const postListener = async (req, res) => {
...
+        try {
+          let fileInfo = JSON.parse(fields["fields"]);
+          //Send file process request to NATS
+          let processingReq = {
+            uploaded_path: uploadDestPath,
+            filename: fileInfo.original_filename,
+          };
+
+          let natsConn = await createNatsConn();
+          await pubFileProcessingReq(natsConn, processingReq);
```
Comment (adev4a) [inline telematic_system/telematic_apps/web_app/server/file_upload_server.js:127]: Now that we have added the nats request subscription on a separate thread for the rosbag_processing service, can we check for a response and print it to screen?

File: telematic_system/telematic_apps/web_app/client/src/components/ros2_rosbag/ROS2RosbagMetadata.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ * Description: Metadata used by other components
+ */
+export const UPLOAD_STATUS = {
+  NA: "NA",
+  IN_PROGRESS: "IN_PROGRESS",
+  ERROR: "ERROR",
+  COMPLETED: "COMPLETED",
+};
+
+export const PROCESSING_STATUS = {
```
[the remark this answers is not in the document]
Comment (dan-du-car) [reply telematic_system/telematic_apps/web_app/client/src/components/ros2_rosbag/ROS2RosbagMetadata.js:25]: This is the filter dropdown to filter files by processing status.

File: telematic_system/telematic_apps/web_app/server/file_upload_server.js (hunk 1 above)
Comment (dan-du-car) [reply telematic_system/telematic_apps/web_app/server/file_upload_server.js:127]: yes, the response is printed.

Comment (dan-du-car) [reply telematic_system/telematic_apps/web_app/server/file_upload_server.js:127]: image

Comment (dan-du-car) [reply telematic_system/telematic_apps/web_app/server/file_upload_server.js:127]: image

File: telematic_system/telematic_apps/web_app/client/src/components/ros2_rosbag/ROS2RosbagMetadata.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (dan-du-car) [reply telematic_system/telematic_apps/web_app/client/src/components/ros2_rosbag/ROS2RosbagMetadata.js:25]: image

**Review by adev4a — APPROVED** [review]

**Pull request merged**